README.md

```diff
@@ -49,6 +49,79 @@ redis:
+hvcNAQEBBQADSwAwSAJBANkBa53UGhd9RYiAZPGOz0/Y9P4/o6uHw/Eh4ExgCrpx
+17NNV1JSAQlVnHtVANGmdz9J0c0MWC2ya3o39BbK7/cCAwEAATANBgkqhkiG9w0B
+AQQFAANBACma7rGAI3khftF9du1KwivWzeGPHJwZBMfL/F99d2ckTyQozLTTL/p3
+U1aTnHBR8cl5yTMAD8onBa/j7HhvL/Q=
+-----END CERTIFICATE-----' \
+	-e ICINGADB_REDIS_KEY='-----BEGIN PRIVATE KEY-----
+MIIBVQIBADANBgkqhkiG9w0BAQEFAASCAT8wggE7AgEAAkEA2QFrndQaF31FiIBk
+8Y7PT9j0/j+jq4fD8SHgTGAKunHXs01XUlIBCVWce1UA0aZ3P0nRzQxYLbJrejf0
+Fsrv9wIDAQABAkEAm0xV7MRey9Kd0Vs5Ylm2aUk1w0Jd6iKmCkoZD+9nnhcKSNuR
+Jf3I9OAXYWCOIEszfrFyAQDTdp9UrOyeE9U7SQIhAP7cREMVA0NryBqYwJketN54
+3unUJGBkVeumyXA/EMIFAiEA2fnScmRn4cXqqxe9Dkgn2RiogTkCZ8h5BdY67xta
+nssCIF6gT+QMUDrfMNvXLWNsyED15eYxsxPrDQ/CzHYVpFY1AiEAz080gatQyX+s
+kpB/NCgYDffPuyb3TLFzuMNpRaOkakUCIHtBnos4xywZBqDdRIenbxRdQHX/llUx
+r1WLl8RkIQ3V
+-----END PRIVATE KEY-----' \
+	-e ICINGADB_REDIS_CA='-----BEGIN CERTIFICATE-----
+MIIBAzCBrgIBKjANBgkqhkiG9w0BAQQFADANMQswCQYDVQQDDAI0MjAeFw0yMTA1
+MTcxMDI3MDlaFw0yMTA1MTgxMDI3MDlaMA0xCzAJBgNVBAMMAjQyMFwwDQYJKoZI
+hvcNAQEBBQADSwAwSAJBANkBa53UGhd9RYiAZPGOz0/Y9P4/o6uHw/Eh4ExgCrpx
+17NNV1JSAQlVnHtVANGmdz9J0c0MWC2ya3o39BbK7/cCAwEAATANBgkqhkiG9w0B
+AQQFAANBACma7rGAI3khftF9du1KwivWzeGPHJwZBMfL/F99d2ckTyQozLTTL/p3
+U1aTnHBR8cl5yTMAD8onBa/j7HhvL/Q=
+-----END CERTIFICATE-----' \
+	-e ICINGADB_DATABASE_HOST=mariadb-icingadb \
+	-e ICINGADB_DATABASE_PORT=3306 \
+	-e ICINGADB_DATABASE_DATABASE=icingadb \
+	-e ICINGADB_DATABASE_USER=icingadb \
+	-e ICINGADB_DATABASE_PASSWORD=123456 \
+	-e ICINGADB_DATABASE_TLS=true \
+	-e ICINGADB_DATABASE_CERT='-----BEGIN CERTIFICATE-----
+MIIBAzCBrgIBKjANBgkqhkiG9w0BAQQFADANMQswCQYDVQQDDAI0MjAeFw0yMTA1
+MTcxMDI3MDlaFw0yMTA1MTgxMDI3MDlaMA0xCzAJBgNVBAMMAjQyMFwwDQYJKoZI
+hvcNAQEBBQADSwAwSAJBANkBa53UGhd9RYiAZPGOz0/Y9P4/o6uHw/Eh4ExgCrpx
+17NNV1JSAQlVnHtVANGmdz9J0c0MWC2ya3o39BbK7/cCAwEAATANBgkqhkiG9w0B
+AQQFAANBACma7rGAI3khftF9du1KwivWzeGPHJwZBMfL/F99d2ckTyQozLTTL/p3
+U1aTnHBR8cl5yTMAD8onBa/j7HhvL/Q=
+-----END CERTIFICATE-----' \
+	-e ICINGADB_DATABASE_KEY='-----BEGIN PRIVATE KEY-----
+MIIBVQIBADANBgkqhkiG9w0BAQEFAASCAT8wggE7AgEAAkEA2QFrndQaF31FiIBk
+8Y7PT9j0/j+jq4fD8SHgTGAKunHXs01XUlIBCVWce1UA0aZ3P0nRzQxYLbJrejf0
+Fsrv9wIDAQABAkEAm0xV7MRey9Kd0Vs5Ylm2aUk1w0Jd6iKmCkoZD+9nnhcKSNuR
+Jf3I9OAXYWCOIEszfrFyAQDTdp9UrOyeE9U7SQIhAP7cREMVA0NryBqYwJketN54
+3unUJGBkVeumyXA/EMIFAiEA2fnScmRn4cXqqxe9Dkgn2RiogTkCZ8h5BdY67xta
+nssCIF6gT+QMUDrfMNvXLWNsyED15eYxsxPrDQ/CzHYVpFY1AiEAz080gatQyX+s
+kpB/NCgYDffPuyb3TLFzuMNpRaOkakUCIHtBnos4xywZBqDdRIenbxRdQHX/llUx
+r1WLl8RkIQ3V
+-----END PRIVATE KEY-----' \
+	-e ICINGADB_DATABASE_CA='-----BEGIN CERTIFICATE-----
+MIIBAzCBrgIBKjANBgkqhkiG9w0BAQQFADANMQswCQYDVQQDDAI0MjAeFw0yMTA1
+MTcxMDI3MDlaFw0yMTA1MTgxMDI3MDlaMA0xCzAJBgNVBAMMAjQyMFwwDQYJKoZI
+hvcNAQEBBQADSwAwSAJBANkBa53UGhd9RYiAZPGOz0/Y9P4/o6uHw/Eh4ExgCrpx
+17NNV1JSAQlVnHtVANGmdz9J0c0MWC2ya3o39BbK7/cCAwEAATANBgkqhkiG9w0B
+AQQFAANBACma7rGAI3khftF9du1KwivWzeGPHJwZBMfL/F99d2ckTyQozLTTL/p3
+U1aTnHBR8cl5yTMAD8onBa/j7HhvL/Q=
+-----END CERTIFICATE-----' \
```

Comment:
What's the point of giving some full PEM file each time? I think it just looks cluttered and provides no benefit. Anyone who want's to use it has to delete all the certificates/keys anyways. So why not just state `[PEM-encoded certificate or path to a file mounted as a volume]`?